app/views/api/integrations/apicast/shared/_mapping_rules.html.slim

```diff
@@ -1,40 +1,4 @@
-/ TODO: THREESCALE-3759 independent mapping rules always enabled with APIAP
-- if current_account.independent_mapping_rules_enabled?
-  fieldset.inputs.independent_mapping_rules
-    legend
-      span
-        = link_to 'Mapping Rules', admin_service_proxy_rules_path(@service), target: '_blank'
-- else
-  = f.toggled_inputs 'mapping_rules', cookie_name: 'mapping-rules' , legend: 'Mapping Rules', id: "mapping-rules" do
-
-    table#proxy-rules
-      thead
-        tr
-          th Verb
-          th Pattern
-          - if @service.using_proxy_pro?
-            th Redirect
-          th title="increment" +
-          th
-            ' Metric or Method
-            | (
-            = link_to 'Define', admin_service_metrics_path(@service), title: 'Define Methods & Metrics for this service'
-            | )
-          th colspan="2"
-            ' Last?
-
-      tbody#sortable.ui-sortable
-        - metrics = options_from_collection_for_select(@service.metrics, 'id', 'name')
-
-        - @proxy.proxy_rules.ordered.each_with_index do |rule, i|
-          tr.ui-sortable-item
-            = render 'api/integrations/apicast/shared/mapping_rule', form: f, rule: rule, metrics: metrics, index: (i + 1)
-        tr#new-proxy-rule-template.ui-sortable-item style="display:none;"
-          = render 'api/integrations/apicast/shared/mapping_rule', form: f, rule: @proxy.proxy_rules.new(delta: 1), disabled: true, metrics: metrics
-      tfoot
-        tr
-          th colspan=(@service.using_proxy_pro? ? '7' : '6')
-            = link_to "Add Mapping Rule", '#add-proxy-rule', class: 'action add', id: 'add-proxy-rule'
-
-- content_for :javascripts do
-  = javascript_include_tag 'mapping_rules'
```

Comment:
Why removing this? it was not inside the `else` 😕

Reply:
Wait, you may be right here, @didierofrivia should review this since he did https://github.com/3scale/porta/pull/530

In that case, it means:
1. The sorting by dragging was working only for the page of non-APIAP. Introduced in 2.5 in [THREESCALE-1787](https://issues.redhat.com/browse/THREESCALE-1787). But we have the new page of APIAP since 2.7 and it may not work by drag and drop.
2. Mystery solved about the 🥒  `Scenario: Sorting mapping rules`... it shouldn't work since it is the APIAP page, so the solution should be removing the cucumber.
3. We should consider if we want to implement the drag and drop for APIAP. If so, out of the scope of this PR, of course.

Reply:
This `content_for :javascript` used to be inside `else`. It was wrongly indented in this commit https://github.com/3scale/porta/commit/56ad25225cd115e36b21dfef2d5653a0ff79ea6a#diff-32cff577d8401c8cfb608cfe94d5643ba1f66497a9412a683635e1aad801c249L39-R40.